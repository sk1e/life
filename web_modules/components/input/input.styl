json('./constants.json')

.input
  width 6rem
  font-size 1rem
  border none
  border-radius 0.1rem
  padding 0.4rem
  height input-height
  background #333
  color #eee