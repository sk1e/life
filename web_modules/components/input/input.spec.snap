exports[`Input renders correctly`] = `
<input
  className="input"
  onBlur={[Function onBlur]}
  placeholder="" />
`;
